### PR TITLE
DTS (Levelbuilder > Staging) reserialize script json to add back level keys

### PR DESCRIPTION
**Pull request by davidsbailey**

### CODE REVIEW

**Review by bethanyaconnor — APPROVED** [review]
What command did you use to regenerate all of these? 

Also is there any risk that there are any script changes that shouldn't be deployed yet? I think no based on Suresh's email that didn't list any script_json files as staged on levelbuilder but wanted to double check

**Comment by davidsbailey** [comment]
> What command did you use to regenerate all of these?

```
Script.all.each(&:write_script_json)
```

> Also is there any risk that there are any script changes that shouldn't be deployed yet? I think no based on Suresh's email that didn't list any script_json files as staged on levelbuilder but wanted to double check

I did verify that there are no content changes in this PR. I will double-check before merging that no new changes have been pushed to the levelbuilder branch. since I am just merging levelbuilder -> staging, this PR would not discriminate between the changes I am trying to pull in and anything else that got pushed to the levelbuilder branch before I merge.

Apologies for not providing a proper PR description 😬

**Review by dmcavoy — APPROVED** [review]
Thank you for doing this and getting it approved. It will help a lot

File: dashboard/config/scripts_json/course2.script_json
Comment (davidsbailey) [inline dashboard/config/scripts_json/course2.script_json:5236]: it looks like what's happening here is that the two `levels_script_levels` objects are swapping positions, because we started sorting level keys a while back and that caused these two lines to swap positions: https://github.com/code-dot-org/code-dot-org/blob/830608c48ee001e9ef70f45f7dd06238eade1451/dashboard/config/scripts_json/course2.script_json#L1401-L1404

however, the thing that controls which of the two levels_script_levels gets shown to the user is the `variants` property: https://github.com/code-dot-org/code-dot-org/blob/830608c48ee001e9ef70f45f7dd06238eade1451/dashboard/config/scripts_json/course2.script_json#L1388-L1395 

so I think we are good here. I've also verified locally that I still see `2-3 Artist 9 NEW` when navigating to this script level: 
Screen Shot 2021-11-22 at 10 43 49 AM

**Comment by davidsbailey** [comment]
To write down some answers to earlier questions from the review meeting:

* this PR could make PDF generation take ~2 hours again on staging. I offered to try and pull out any lines in this PR which might not be necessary, such as coursea-2021.script_json in which only serialized_at is modified, and the response from the room was that if this only takes 2 hours the very next time that the staging build runs then it should be ok in terms of not holding up our deploy today.

**Pull request merged**
